{
 "PRed": [
  {
   "PR": {
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python310"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 3,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "1.1.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 4,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2022.11.05.21.08.08",
 "pre_pr_migrator_attempts": {
  "pypy38": 5
 },
 "pre_pr_migrator_status": {
  "pypy38": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/5664406631\">bot CI job</a>): main: ['linux_64_python3.8.____73_pypy: Encountered problems while solving:\\n  - package pip-8.1.2-py34_0 requires python 3.4*, but none of the providers can be installed', 'linux_64_python3.8.____73_pypy: Encountered problems while solving:\\n  - package pypy3.8-7.3.11-h75e580f_0 requires tk >=8.6.12,<8.7.0a0, but none of the providers can be installed', 'linux_64_python3.9.____73_pypy: Encountered problems while solving:\\n  - package pip-8.1.2-py34_0 requires python 3.4*, but none of the providers can be installed', 'linux_64_python3.9.____73_pypy: Encountered problems while solving:\\n  - package pypy3.9-7.3.11-h527bfed_0 requires tk >=8.6.12,<8.7.0a0, but none of the providers can be installed', 'osx_64_python3.8.____73_pypy: Encountered problems while solving:\\n  - package python-htcondor-9.0.6-py38h41b476c_2 requires python_abi 3.8.* *_cp38, but none of the providers can be installed', 'osx_64_python3.9.____73_pypy: Encountered problems while solving:\\n  - package python-htcondor-9.0.6-py39hc94b615_2 requires python_abi 3.9.* *_cp39, but none of the providers can be installed']"
 },
 "smithy_version": "3.21.3"
}